src/server/qgssldconfigparser.cpp

```diff
@@ -724,20 +724,11 @@ bool QgsSLDConfigParser::wmsInspireActivated() const
   return false;
 }
 
-QgsComposition* QgsSLDConfigParser::createPrintComposition( const QString& composerTemplate, QgsMapRenderer* mapRenderer, const QMap< QString, QString >& parameterMap, QStringList& highlightLayers ) const
+QgsComposition* QgsSLDConfigParser::initComposition( const QString& composerTemplate, QgsMapSettings* mapSettings, QList< QgsComposerMap*>& mapList, QList< QgsComposerLegend* >& legendList, QList< QgsComposerLabel* >& labelList, QList<const QgsComposerHtml *>& htmlFrameList ) const
```

Comment:
I'd make the mapSettings argument a reference here - it's required and composer will crash if a nullptr is passed.